config/config-example.js

```diff
@@ -210,6 +210,11 @@ exports.appealurl = '';
 exports.replsocketprefix = './logs/repl/';
 exports.replsocketmode = 0o600;
 
+// ripgrepmodlog - whether to use ripgrep to handle modlog queries
+// If no value is set, the first modlog query after each restart will check if
+// ripgrep binaries are available and, if so, they will be used
+exports.ripgrepmodlog = undefined;
```

Comment:
In general, only config settings that a server operator is likely to care about should be in `config-example`.